h/storage.py

```diff
@@ -177,6 +170,10 @@ def update_annotation(session, id_, data):
     :param data: the validated data with which to update the annotation
     :type data: dict
 
+    :param group_service: a service object that implements
```

Comment:
IMO the `:param` docstring's content is redundant here. If the purpose of a parameter is clear from the type, then I think it is fine to omit the description.